cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -58,6 +58,7 @@ spec:
         # command = "/skydns"
         - -machines=http://localhost:4001
         - -addr=0.0.0.0:53
+        - -rtimeout=1s
```

Comment:
We will re-adjust / remove this flag if/when we pick the latest skydns release.

Also, hopefully, we will stop using skydns as a forwarder, making this flag redundant.